app/src/main/java/com/rage/clamber/Activities/LoginActivity.java

```diff
@@ -31,17 +49,85 @@ protected void onCreate(Bundle savedInstanceState) {
     //TODO: submit new user button will update external database with new user.
     //TODO: submit existing user button will call external database and update app with users data.
 
-    @OnClick(R.id.login_activity_submit_new_user_button)
+    @OnClick(R.id.login_activity_new_user_button)
     public void onNewUserButtonClicked (Button button) {
-        Intent intent = new Intent(this, HomePage.class);
-        startActivity(intent);
+        UserInfoDialogFragment fragment = new UserInfoDialogFragment();
+        fragment.show(getSupportFragmentManager(), "dialog");
+
     }
 
-    @OnClick(R.id.login_activity_submit_already_user_button)
+    @OnClick(R.id.login_activity_existing_user_button)
     public void onExistingUserButtonClicked (Button button) {
-        Intent intent = new Intent(this, HomePage.class);
-        startActivity(intent);
+        ExistingUserLoginFragment fragment = new ExistingUserLoginFragment();
+        fragment.show(getSupportFragmentManager(), "dialog");
+
+    }
+
+    public void existingUserPositiveClick(String user) {
+        userName = user;
+        if (userName.length() == 0) {
```

Comment:
good point - from looking around online it looks like I should be able to do it like this, but I should also trim it in case a space is entered. I'll update it to do that and test and re-commit.